apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsSocialMediaControl/constants.tsx

```diff
@@ -0,0 +1,90 @@
+import type { FooterSchemaType } from "@opengovsg/isomer-components"
+import { Icon } from "@chakra-ui/react"
+import { AiFillTikTok } from "react-icons/ai"
+import {
+  FaFacebook,
+  FaGithub,
+  FaLinkedin,
+  FaSquareXTwitter,
+  FaTelegram,
+  FaThreads,
+  FaYoutube,
+} from "react-icons/fa6"
+
+import { IconFlickr, IconInstagram, IconWhatsApp } from "~/components/icons"
```

Comment:
curious: why custom SVG instead of react-icons?

Reply:
Bopes this one, react-icons are single coloured but these 3 icons are multi-coloured, so need to use their original icons